services/brig/brig.cabal

```diff
@@ -16,8 +16,6 @@ copyright:      (c) 2017 Wire Swiss GmbH
 license:        AGPL-3
 license-file:   LICENSE
 build-type:     Custom
-data-files:
-    docs/swagger.md
```

Comment:
don't files that are required for compilation also count as data files?

Reply:
I don't think so.

Reply:
but they should make it into the tar ball created by `cabal sdist`, no?  and that's only happening if they are listed here.

```
$ cabal sdist
remote: Enumerating objects: 60, done.
remote: Counting objects: 100% (60/60), done.
remote: Compressing objects: 100% (25/25), done.
remote: Total 60 (delta 40), reused 55 (delta 35), pack-reused 0
Unpacking objects: 100% (60/60), 14.09 KiB | 721.00 KiB/s, done.
From https://github.com/wireapp/http2
 * [new branch]      aeson-2             -> origin/aeson-2
 * [new branch]      client-status-fixme -> origin/client-status-fixme
 * [new branch]      preface-race        -> origin/preface-race
 * [new branch]      smatting/debugging  -> origin/smatting/debugging
Previous HEAD position was 7c465be Better support for setting header table size to 0
HEAD is now at 1ee1ce4 Make sure connection preface is always sent first
cabal: Error: Could not find module: Brig.Docs.Swagger with any suffix:
["gc","chs","hsc","x","y","ly","cpphs","hs","lhs","hsig","lhsig"]. If the
module is autogenerated it should be added to 'autogen-modules'.
```

ah, the last line is interesting!

Reply:
(yes, we don't do sdists, but we might as well pretend! :))